com.github.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
```

Comment:
Are you sure you need the GNOME platform? Freedesktop-sdk ought to be enough

Reply:
How can I check if it needs it or not? Fluffychat uses it so do I, also flutter apps uses libgtk3-devel to compile so I don't know much about this runtime.